clang/test/AST/HLSL/resource_binding_attr.hlsl

```diff
@@ -34,6 +34,10 @@ RWBuffer<float> UAV1 : register(u2), UAV2 : register(u4);
 // CHECK: HLSLResourceBindingAttr {{.*}} "" "space5"
 RWBuffer<float> UAV3 : register(space5);
 
+// CHECK: VarDecl {{.*}} UAV_Array 'RWBuffer<float>[10]'
```

Comment:
Should we add a test case where HLSLVkBindingAttr already exists (I presume an explicit binding case), we check for HLSLVkBindingAttr, and check NOT that the new attr is added?

Reply:
Will do.